c/include/libsbp/sbp.h

```diff
@@ -87,6 +87,8 @@ void sbp_clear_callbacks(sbp_state_t* s);
 void sbp_state_init(sbp_state_t *s);
 void sbp_state_set_io_context(sbp_state_t *s, void* context);
 s8 sbp_process(sbp_state_t *s, s32 (*read)(u8 *buff, u32 n, void* context));
+s8 sbp_process_generic(sbp_state_t *s, s32 (*read)(u8 *buff, u32 n, void *context),
```

Comment:
Nit: This is fairly generic name. Maybe something like `sbp_process_msg_catchall` or something like that? As an alternative I think a comment explaining what this is meant for would help. I know we don't have any comments for the other functions, but they are sorta-kinda self explanatory while this one isn't.

Reply:
I renamed to sbp_process_all and added some comments per your suggestion.